extras/liquibase-unlocker/pom.xml

```diff
@@ -69,7 +54,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
                 <configuration>
-                    <dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml</dependencyReducedPomLocation>
+                    <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
This option is not inherited from the root pom.xml after merging #3706 so the option can be removed